config_sample/areas.yaml

```diff
@@ -1,9 +1,26 @@
 - area: Basement
   background: gs4
   bglock: true
+  evidence_mod: FFA
+  locking_allowed: false
+  iniswap_allowed: true
```

Comment:
sure
`
if 'evidence_mod' not in item:
                item['evidence_mod'] = 'FFA'
            if 'locking_allowed' not in item:
                item['locking_allowed'] = False
            if 'iniswap_allowed' not in item:
                item['iniswap_allowed'] = True
`